Description: Upstream changes introduced in version 2.0.3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 gbonds (2.0.3-1) unstable; urgency=low
 .
   [ Richard Laager ]
   * New upstream version
   * Remove the Encoding key from the gbonds.desktop file as it is deprecated
   * Change debian/copyright to the DEP5 machine-readable format
   * Document the upstream copyright years in debian/copyright
   * Document the existence of public domain files in debian/copyright
   * Add a Homepage field to debian/control
   * Removed snaught.com from the download list, as it didn't have the latest
     redemption data.  This leaves only the official U.S. Treasury FTP site.
   * Split the arch-indep data out into gbonds-data.
   * Update the standards version to 3.8.4
   * Switch to debhelper 7 with a minimal rules file
   * Switch to source format "3.0 (quilt)"
   * Rerun the autotools because ltmain.sh is ancient
   * Added savings bond redemption data through 11/2010 (sb201006.asc)
   * Disabled printing, as libgnomeprint is going away (Closes: 542565)
 .
   [ tony mancill ]
   * debian/control:
     - Add libtool, intltool, gnome-common to Build-Depends:
   * Add patch debian/patches/POTFILES.missing to allow build.
   * Update missing information in debian/copyright.
     Thank you to Alexander Reichle-Schmehl for his assistance.
 .
 The person named in the Author field signed this changelog entry.
Author: tony mancill <tmancill@debian.org>
Bug-Debian: http://bugs.debian.org/542565

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ gbonds-2.0.3/po/POTFILES.skip
@@ -0,0 +1 @@
+.pc/libgnomeprint/src/commands.c
